### PR TITLE
Add introduction

### PR DESCRIPTION
**Pull request by 17-09**
Leaders don’t create followers, they create more leaders.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +74,7 @@ Add additional notes about how to deploy this on a live system
 
 ## Built With
 
-* [Dotnet Core 3.0](https://dotnet.microsoft.com/download)
+* [Dotnet Core 3.1](https://dotnet.microsoft.com/download)
```
Comment (9-9-9-9) [inline README.md]: Hi, we are using dotnet core 3.0, please kindly revert this change

File: README.md [hunk 2]
```diff
@@ -2,10 +2,18 @@ Code coverage: 50%
...
+# SimpleAuth (Simple authentication & authrozization)
+## A lightweight Authorization server based on RBAC
 
-// TODO
+## Motivation:
+Designing an AA (Authentication & Authorization) from scratch is a costly, repetitive and time-consuming task. OAuth2 and OpenID connect seem like a very reasonable solution for centralize AA in our industry. However it's very complex and working with it is really stressful when you are just starting out with new ideas. 
+Simple Auth is a simple solution, near zero-configuration, easy idea (RBAC).
+
+### Features
+- Persistence of your choice (Postgres, sql lite, in memory, etc).
```
Comment (9-9-9-9) [inline README.md]: This feature is not the thing should stays in the top lines

File: README.md [hunk 3]
```diff
@@ -2,10 +2,18 @@ Code coverage: 50%
...
+- Authorization APIs. (Core interfaces of SimpleAuth).
+- SDKs for zero integration (Currently support .NET Core)
```
Comment (9-9-9-9) [inline README.md]: Kindly add one more thing that supporting for other programming languages would be available in the future.

File: README.md [hunk 4]
```diff
@@ -2,10 +2,18 @@ Code coverage: 50%
...
+- Built-in external authentication providers (Facebook, Google, Linkedin, etc)
```
Comment (9-9-9-9) [inline README.md]: Currently support Google OAuth token

File: README.md [hunk 5]
```diff
@@ -2,10 +2,18 @@ Code coverage: 50%
 
 #### I know 50% is too little, but the core part, the service one, is tested
 
-# SimpleAuth
-## A lightweighted Authorization server, integrated with external Authentication providers
+# SimpleAuth (Simple authentication & authrozization)
```
Comment (9-9-9-9) [inline README.md]: The is a TODO in another branch of you, that is
```
// TODO: Motivation & The reason behind
```
Please kindly resolve this within this Pull request, I will delete your previous branch

File: README.md (hunk 1 above)
Comment (17-09) [reply README.md]: ### is that needed bro? Let's do migrate.

Comment (9-9-9-9) [reply README.md]: I didn't install 3.1 yet

File: README.md (hunk 2 above)
Comment (17-09) [reply README.md]: Pls do it for me :))

Comment (17-09) [reply README.md]: whatever you want

Comment (9-9-9-9) [reply README.md]: I'm a friend, a student of you, not your pet

File: README.md (hunk 3 above)
Comment (17-09) [reply README.md]: Ok

File: README.md (hunk 1 above)
Comment (9-9-9-9) [reply README.md]: I will update this tiny details once I updated my old man laptop

Comment (17-09) [reply README.md]: LOL

File: README.md (hunk 3 above)
Comment (9-9-9-9) [reply README.md]: Hello, I didn't see any change of this line

File: README.md (hunk 1 above)
Comment (9-9-9-9) [reply README.md]: Unresolved
